usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/CatalogItemSummary.java

```diff
@@ -132,6 +157,7 @@ public boolean isDeprecated() {
     @Override
     public String toString() {
         return Objects.toStringHelper(this)
+                .add("tags", tags)
```

Comment:
In the spirit of keeping things concise I'd remove it but it's a minor one. Keep it if you feel strongly.